change/@ni-nimble-angular-761cf0de-3455-4fb8-af25-0631d706dee5.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
The Lighthouse CI status check is showing a new warning. I compared against [a build from another PR](https://storage.googleapis.com/lighthouse-infrastructure.appspot.com/reports/1670885421760-64609.report.html) and didn't notice any dramatic changes, but there was a slight uptick in the number of DOM elements and the "Reduce Unused JavaScript" check.

I'm guessing we were close to a limit before and this just pushed us over the edge but we should talk about what to do. Maybe consider splitting the example app into multiple pages? (This could be follow up tech debt)

Reply:
The kitchen sink examples (the angular and blazor apps) are published on GitHub pages. Using github pages requires using hash-based routing. Angular supports that but blazor does not. So that example cannot be split into different pages (we try and keep the changes in-sync between them). I would not recommend doing that.

Reply:
After discussion in a team meeting we decided that we are ok with this check returning a warning for now. We don't want to split up the "kitchen sink" example pages so some of the lighthouse checks like "too many DOM elements" will naturally degrade. But the check still provides some value in showing us drastic regressions.

We'll re-evaluate when we come up with a broader performance testing strategy in #614